api/api-doc/storage_service.json

```diff
@@ -2324,6 +2359,103 @@
                "description":"The endpoint details"
             }
          }
+      },
+      "map_entry":{
```

Comment:
we already have a model call `mapper` that has the exact same key/value attributes

Reply:
Missed that.

Reply:
fixed